src/test/java/HW/HW3_1.java

```diff
@@ -0,0 +1,185 @@
+package HW;
+
+import Class1.practice;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+public class HW3_1 {
+
+    @Test
+    public void CorrectTemp() {
+        practice.openURL("https://www.darksky.net/");
+        String taketempnow = "//span[@class='summary swap']";
+        By tempnowLocator = By.xpath(taketempnow);
+        WebElement tempnowElement = practice.getDriver().findElement(tempnowLocator);
+        String curTempfar = tempnowElement.getText();
+
+        String currentTempFar = "//div[@id='header']/div[1]/div[1]/div[2]/span[1]";
+        By farLocator = By.xpath(currentTempFar);
+        WebElement farElement = practice.getDriver().findElement(farLocator);
+        farElement.click();
+
+        practice.sleep(2);
+
+        String c_xpa = "//div[@id='header']//li[starts-with(text(),'˚C')][1]";
+        By cLocator = By.xpath(c_xpa);
+        WebElement change = practice.getDriver().findElement(cLocator);
+        change.click();
+
+        String xPathTemp = "//span[@class='summary swap']";
+        By cLocatorTemp = By.xpath(xPathTemp);
+        WebElement cElement = practice.getDriver().findElement(cLocatorTemp);
+        String ctemp = cElement.getText();
+        Assert.assertEquals(ctemp, "2˚ Clear.", "temperature changed");
```

Comment:
"2˚ Clear." This hard coded verification in this case isn't good at all. It'll be changing every moment.
Again I think as per assignment, you supposed to grab F temp from darksky and convert it to C temp then grab C temp from Darksky; then verify both C temp is equal.